parlai/mturk/core/react_server/dev/components/core_components.jsx

```diff
@@ -522,6 +526,26 @@ class LeftPane extends React.Component {
   }
 }
 
+class LeftPane extends React.Component {
+  render () {
+    let v_id = this.props.v_id;
```

Comment:
i notice that you call this `v_id` but elsewhere it's `agent_id`, does this mean something like `viewer_id`?

Reply:
Yes - `v_id` is like the MTurk equivalent of viewer context. When on MTurk it will always be agent_id, but in the react frontend in order to demo tasks you need to be able to swap between multiple `v_id`'s